exercises/practice/game-of-life/.meta/config.json

```
@@ -0,0 +1,20 @@
+{
+  "authors": [
+    "akbatra567"
+  ],
+  "files": {
+    "solution": [
+      "src/main/java/GameOfLife.java"
+    ],
+    "test": [
+      "src/test/java/GameOfLifeTest.java"
+    ],
+    "example": [".meta/src/reference/java/GameOfLife.java"],
```

I would change `example` to the following to keep it consistent with our current formatting

```suggestion
    "example": [
      ".meta/src/reference/java/GameOfLife.java"
    ],
```

---

I believe the Configlet also complains about this in the CI build 😉